### PR TITLE
WM4 - Oguzhan Bozkurt - JavaScript-Core-1-Coursework-Week1

### PR DESCRIPTION
**Pull request by ozibozkurt**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

-----
[View rendered exercises/G-numbers/README.md](https://github.com/ozibozkurt/JavaScript-Core-1-Coursework-Week1/blob/Oguzhan/exercises/G-numbers/README.md)

### CODE REVIEW

File: exercises/L-functions-nested/exercise.js [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
-var mentor5 = "Yohannes";
+var mentor2 = "Chizim";
+var mentor3 = "Biruk";
+var mentor4 = "Louis";
+var mentor5 = "Ali";
+var mentor6 = "Marta";
+var mentor7 = "Elizabeth";
+var mentor8 = "Huma";
+var mentor9 = "Anna";
+var mentor10 = "Matt";
```
Comment (nholdsworth94) [inline exercises/L-functions-nested/exercise.js:11]: These variables are not being used - do they need to be in here?
Should they perhaps be being used to call the function?

File: exercises/D-strings/exercise.js [hunk 2]
```diff
@@ -1,3 +1,9 @@
 // Start by creating a variable `message`
 
+
+var name = "This is a string";
+var message = typeof name;
+
 console.log(message);
```
Comment (nholdsworth94) [inline exercises/D-strings/exercise.js:7]: take a look at the instructions again - you're missing a small bit :)

File: exercises/F-strings-methods/exercise2.js [hunk 3]
```diff
@@ -1,3 +1,7 @@
-const name = " Daniel  ";
+const name = "          Daniel             ";
+var nameLength = name.length;
+let message = "Hello, my name is ";
+
+console.log(message.trim() + name.trim() + " and my name is " + nameLength + " characters long");
```
Comment (nholdsworth94) [inline exercises/F-strings-methods/exercise2.js:5]: as a result of doing `message.trim()` what is now missing between `message` and `name` in your `console.log`?

File: exercises/G-numbers/exercise.js [hunk 4]
```diff
@@ -1 +1,9 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var NumberOfMentors= 8;
```
Comment (nholdsworth94) [inline exercises/G-numbers/exercise.js:3]: me being picky, but JS variables should be camel case
i.e. they should start with a lower case letter

File: exercises/I-floats/exercise.js [hunk 5]
```diff
@@ -1,2 +1,7 @@
-var numberOfStudents = 15;
-var numberOfMentors = 8;
+const numberOfTrainees = 15;
+const numberOfMentors = 8;
+const total = numberOfTrainees + numberOfMentors;
+const pOfTrainees = (numberOfTrainees / total) * 100;
```
Comment (nholdsworth94) [inline exercises/I-floats/exercise.js:4]: me being picky again - `pOfTrainees` could be a bit more descriptive

File: exercises/L-functions-nested/exercise.js [hunk 6]
```diff
@@ -1,5 +1,18 @@
+function greetings(mentor) {
```
Comment (nholdsworth94) [inline exercises/L-functions-nested/exercise.js:1]: if you re-read the instructions, it is asking for **2** functions here
1 to change the string to upper case
1 to create a shouty greeting
whilst what you have done does work, it isn't what the exercise is asking for

File: mandatory/3-function-output.js [hunk 7]
```diff
@@ -2,16 +2,22 @@
 function getRandomNumber() {
   return Math.random() * 10;
 }
+console.log(getRandomNumber);
+//this method  will return a number between 0 and 9
+
 
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
```
Comment (nholdsworth94) [inline mandatory/3-function-output.js:10]: no comments added here to explain function

File: mandatory/3-function-output.js [hunk 8]
```diff
@@ -2,16 +2,22 @@
 function getRandomNumber() {
```
Comment (nholdsworth94) [inline mandatory/3-function-output.js:2]: no comments added here to explain function

**Review by nholdsworth94 — COMMENTED** [review]
For the most part, this looks good Ozzy
well done :D

**Comment by ozibozkurt** [comment]
> For the most part, this looks good Ozzy well done :D

Thank you for your feedbacks, I fixed all my errors:)

**Comment by ozibozkurt** [comment]
Thank you for your helping Nick.

**Review by nholdsworth94 — COMMENTED** [review]
Thanks for coming back to this, Ozzy. Great work!

**Pull request closed without merging**